package com.hninor.pruebainterrapidisimo.core

class DataException(val customMessage: String) : Exception()
